process/headerCheck/headerSignatureVerify.go

```diff
@@ -89,10 +89,16 @@ func (hsv *HeaderSigVerifier) VerifySignature(header data.HeaderHandler) error {
 		return process.ErrBlockProposerSignatureMissing
 	}
 
+	epoch := header.GetEpoch()
+	if header.IsStartOfEpochBlock() && epoch > 0 {
+		epoch = epoch - 1
+	}
```

Comment:
Is this related to the new Validators versus old Validators for the first Consensus of the Epoch?

Reply:
Yes, this is currently a workaround, which uses the old epoch validators to validate the new epoch first block. Should finally be changed to have the block validated by the new epoch validators.